README.rst

```diff
@@ -1,7 +1,44 @@
 Byterun
 -------
 
-This is a pure-Python implementation of a Python bytecode execution virtual
-machine.  I started it to get a better understanding of bytecodes so I could
-fix branch coverage bugs in coverage.py.
+
+
+Byterun is a python interpreter written in just 1451 lines of Python.
+Byterun's goal is to clearly explain Python's design. 
+If you want to better understand Python, or you are not quite sure how something in cPYthon works, then Byterun is  a great place to start. 
+Byterun has proven very useful to a number of different projects, `including one by Google <https://github.com/nedbat/byterun/pull/12>`_.  
```

Comment:
I think this may promote the wrong mindset about Byterun. Byterun is a great teaching and learning tool but is not great for actually executing CPython bytecode because performance is not a key concern.